.github/workflows/php.yml

```
@@ -42,7 +42,7 @@ jobs:
         run: COMPOSER_PROCESS_TIMEOUT=600 composer install --ansi --prefer-dist --no-interaction --no-progress
 
       - name: Run PHPCSFixer
-        run: ./vendor/bin/php-cs-fixer fix --dry-run --diff
+        run: ./vendor/bin/php-cs-fixer fix --dry-run --diff --path-mode=intersection
```

```suggestion
        run: ./vendor/bin/php-cs-fixer fix --dry-run --diff
```
This part was improved since 8.2.x The problem came from the config, with the config fixed there is no need for this parameter anymore